usuarios/views.py

```diff
@@ -22,6 +23,16 @@ class CreateView(crud.base.CrudCreateView):
                               validação do seu perfil.'
 
         def get_success_url(self):
```

Comment:
Uma idéia seria utilizar signals do Django. Veja 

https://coderwall.com/p/ktdb3g/django-signals-an-extremely-simplified-explanation-for-beginners

http://stackoverflow.com/questions/15563413/django-using-custom-signals-to-send-mail